VultisigApp/VultisigApp/Chains/Ton.swift

```diff
@@ -43,14 +43,19 @@ enum TonHelper {
             $0.amount = UInt64(keysignPayload.toAmount.description) ?? 0
             $0.mode = sendMode
             
+            // Set bounceable based on chainSpecific parameter
```

Comment:
@enriquesouza , when `sendMaxAmount` is true , do we need to override the amount to 0? 
since the sendMode will be `attachAllContractBalance`